test/integration/v3000-py2/controls/service_spec.rb

```diff
@@ -1,12 +1,19 @@
 # frozen_string_literal: true
 
+services =
+  case platform[:family]
```

Comment:
Should it be `system.platform[:family]` if at some point in the future we need to do special stuffs for `arch` or `gentoo` and be consistent with other places?

Reply:
@baby-gnu I suppose my thought pattern had been to use pure InSpec as much as possible and only rely on our library when necessary.  Anyway, I've taken your suggestion and applied it -- thanks for that.